src/chain/tezos/contracts/tzip12/MultiAssetTokenHelper.ts

```diff
@@ -62,21 +67,17 @@ export namespace MultiAssetTokenHelper {
-        const storage = `( Pair ( Pair "${administrator}" ( Pair 0 { } ) ) ( Pair ( Pair Unit { } ) ( Pair ${pause ? 'True' : 'False'} { Elt ${tokenid} ( Pair ( Pair ${tokenid} ( Pair "${symbol}" ( Pair "${name}" ( Pair ${scale} { } ) ) ) ) ${supply} ) } ) ) )`;
-
+    export async function deployContract(server: string, signer: Signer, keystore: KeyStore, fee: number, administrator: string, metadataUrl: string, pause: boolean = true, gas: number = 800_000, freight: number = 20_000): Promise<string> {
```

Comment:
Specify and link to the (presumably) smartpy code that was used to generate this.